### PR TITLE
purge fix

### PR DESCRIPTION
**Pull request by czechbol**
fixed a couple of issues. Needs sk.popie

### CODE REVIEW

File: purge/module.py [hunk 1]
```diff
@@ -32,21 +31,80 @@ async def purge(self, ctx: commands.Context, num_to_delete: int = None):
...
                 return
             replied_to = msg.reference.cached_message
             if replied_to is None:
                 replied_to = await ctx.fetch_message(msg.reference.message_id)
-            time = replied_to.created_at
-            deleted = await channel.purge(after=time, check=self._not_pinned)
+
+            embed = utils.discord.create_embed(
+                author=ctx.author,
+                title="Confirm delete.",
```
Comment (sinus-x) [inline purge/module.py:48]: Let's make this translatable, too.

File: purge/module.py [hunk 2]
```diff
@@ -32,21 +31,80 @@ async def purge(self, ctx: commands.Context, num_to_delete: int = None):
...
+                )
+            else:
+                await ctx.send(_(ctx, "Aborted."))
+                return
+
+        elif count is not None:
+            if count > 10:
+                embed = utils.discord.create_embed(
+                    author=ctx.author,
+                    title="Confirm delete.",
```
Comment (sinus-x) [inline purge/module.py:76]: ditto

**Comment by czechbol** [comment]
> The confirmation dialog bumped it up a lot. But that's fine, as this is the only functionality in this module. Better safe than sorry.

Addressed comments, added sk translations

**Review by sinus-x — APPROVED** [review]
LGTM now

**Pull request merged**